css/style.css

```diff
@@ -394,18 +395,19 @@ nav .fa {
 
 .card {
   cursor: pointer;
-  background: rgb(1, 59, 112);
+  background: rgb(5, 66, 123);
   box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
   text-align: center;
-  transition: 0.3s ease-in-out;
+  transition: 0.4s ease-in-out;

```

Comment:
If you want to code like a professional, there is no need like this:

```diff
- transition: 0.3s ease-in-out;
+ transition: .3s ease-in-out;
```
Overall looking Great.